src/web/lib/ArticleRenderer.tsx

```diff
@@ -153,15 +153,19 @@ export const ArticleRenderer: React.FC<{
                     return <TableBlockComponent element={element} />;
                 case 'model.dotcomrendering.pageElements.TextBlockElement':
                     return (
-                        <TextBlockComponent
-                            key={i}
-                            isFirstParagraph={i === 0}
-                            html={element.html}
-                            pillar={pillar}
-                            display={display}
-                            designType={designType}
-                            forceDropCap={element.dropCap}
-                        />
+                        <>
+                            <TextBlockComponent
+                                key={i}
+                                isFirstParagraph={i === 0}
+                                html={element.html}
+                                pillar={pillar}
+                                display={display}
+                                designType={designType}
+                                forceDropCap={element.dropCap}
+                            />
+                            {/* Insert the placeholder for the sign in gate on the 2nd paragrah */}
+                            {i === 1 && <span id="sign-in-gate" />}
```

Comment:
Ideally we'd move this to where ads are sliced in - let me get back to you on that as I can't find it and want to get the rest of the comments posted!

Reply:
I'll come back to this, on rota this week so slammed, but will put it on my list to refactor as it's a bit of an architectural thing!